tICA/tICAPipeline.sh

```diff
@@ -149,6 +160,25 @@ then
     signalTxtName="ReCleanSignal.txt"
 fi
 
+if [[ "$IsLongitudinal" == "1" ]]; then
```

Comment:
Consider similar boolean style (to match what you already did with `IsLongitudinal` at the end of the script).
```suggestion
if ((IsLongitudinal)); then
```